src/lib/Microsoft.Health.Fhir.Ingest/Telemetry/ExceptionTelemetryProcessor.cs

```diff
@@ -31,33 +30,71 @@ public ExceptionTelemetryProcessor()
         {
         }
 
-        public ExceptionTelemetryProcessor(params Type[] handledExceptionTypes)
+        public ExceptionTelemetryProcessor(params System.Type[] handledExceptionTypes)
         {
-            _handledExceptions = new HashSet<Type>(handledExceptionTypes);
+            _handledExceptions = new HashSet<System.Type>(handledExceptionTypes);
         }
 
-        public virtual bool HandleException(Exception ex, ILogger log)
+        public virtual bool HandleException(Exception ex, ITelemetryLogger log, string connectorStage)
         {
             EnsureArg.IsNotNull(ex, nameof(ex));
+            EnsureArg.IsNotNull(log, nameof(log));
+
             var exType = ex.GetType();
 
             var lookupType = exType.IsGenericType ? exType.GetGenericTypeDefinition() : exType;
 
             if (_handledExceptions.Contains(lookupType))
             {
-                if (ex is ITelemetryEvent evt)

```

Comment:
This is no longer being used and should be removed.

Reply:
Updated